Documentation/coding-guidelines/breaking-change-rules.md

```diff
@@ -158,16 +158,17 @@ Breaking Change Rules
 * Adding `virtual` to a member  
 > Make note, that marking a member virtual might cause previous consumers to still call the member non-virtually.
 
-* Introducing or removing an override
-> Make note, that introducing an override might cause previous consumers to skip over the override when calling `base`.
-
 &#10007; **Disallowed**
 * Adding an member to an interface
 
 * Adding an abstract member to a type when there _are accessible_ (`public` or `protected`) constructors and the type is not `sealed`
 
 * Adding a constructor to a class which previously had no constructor, without also adding the default constructor
 
+* Introducing or removing an override to non-sealed `protected` or `protected internal` types
+> Introducing an override will cause previous consumers to skip over the override when calling `base` if not compiled against the new version.
+> Removing an override will cause previous consumers that call `base` to throw `TypeLoadException` if not compiled against the new version.
```

Comment:
OOPS! You're absolutely right! Damned copy+paste!

Although `C.GetInfo` is still calling `B.GetInfo` after [line #10](https://github.com/paulomorgado/CSharpVirtualCallGotchas/blob/master/test2.cmd#L10):

```IL
.method public hidebysig virtual instance string 
        GetInfo() cil managed
{
  // Code size       22 (0x16)
  .maxstack  8
  IL_0000:  ldstr      "{0}->{1}"
  IL_0005:  ldarg.0
  IL_0006:  call       instance string [B]B::GetInfo()
  IL_000b:  ldstr      "C"
  IL_0010:  call       string [mscorlib]System.String::Format(string,
                                                              object,
                                                              object)
  IL_0015:  ret
} // end of method C::GetInfo
```

the runtime is cleaver enough to call `A.GetInfo`.

.NET Reflector still complains, though.

It's still bad to remove an overload because, if you later need to reintroduce it, you'll get into to the case of adding an overload. But my premisse was all wrong.